lint/anvil/src/main/java/dev/whosnickdoglio/anvil/detectors/FavorContributesBindingOverBindsDetector.kt

```diff
@@ -39,19 +40,19 @@ internal class FavorContributesBindingOverBindsDetector : Detector(), SourceCode
                     val method = node.uastParent as? UMethod ?: return
                     val returnType = context.evaluator.getTypeClass(method.returnType)
                     // Anvils binding annotations do not support super types that type parameters
-                    // TODO document this better
+                    // https://github.com/square/anvil/pull/726#issuecomment-1657296801
                     if (returnType?.typeParameters?.isEmpty() == true) {
                         if (method.hasAnnotation(INTO_MAP) || method.hasAnnotation(INTO_SET)) {
                             context.report(
                                 Incident(context, ISSUE)
-                                    // TODO try range location
                                     .location(context.getLocation(node.uastParent))
-                                    .message("You can use `@ContributesMultibinding` over `@Binds`"),
+                                    .message(
+                                        "You can use `@ContributesMultibinding` over `@Binds`",
+                                    ),
                             )
                         } else {
                             context.report(
                                 Incident(context, ISSUE)
-                                    // TODO try range location
```

Comment:
Right now it highlights just the method name with the warning instead of the whole binds method (`bindMethod` instead of `@Binds fun bindMethod(impl: Impl): interface`) which is not really what I want but I can come back to that later